base/exports.jl

```diff
@@ -432,37 +422,8 @@ export
     ≉,
 
 # specfun
-    airyai,
-    airyaiprime,
-    airybi,
-    airybiprime,
-    airyaix,
-    airyaiprimex,
-    airybix,
-    airybiprimex,
-    besselh,
-    besselhx,
-    besseli,
-    besselix,
-    besselj,
-    besselj0,
-    besselj1,
-    besseljx,
-    besselk,
-    besselkx,
-    bessely,
-    bessely0,
-    bessely1,
-    besselyx,
     beta,
```

Comment:
I thought about that, but those are simple extensions of `gamma`/`lgamma`, so in the interest of not doing _too_ much at once, I left those for now.